cmake/modules/AddSwift.cmake

```diff
@@ -359,7 +359,7 @@ function(_add_variant_link_flags)
     endif()
   elseif("${LFLAGS_SDK}" STREQUAL "ANDROID")
     list(APPEND result
-        "-ldl" "-llog" "-latomic"
+        "-ldl" "-llog" "-latomic" "-licudata" "-licui18n" "-licuuc"
```

Comment:
Is this change really necessary? 
Now when I run:
$ adb shell LD_LIBRARY_PATH=/data/local/tmp /data/local/tmp/hello

I get:
CANNOT LINK EXECUTABLE: could not load library "libswiftCore.so" needed by "/data/local/tmp/hello"; caused by could not load library "libicudata.so" needed by "libswiftCore.so"; caused by library "libicudata.so" not found